src/main/index.ts

```diff
@@ -184,13 +184,39 @@ app.on("ready", async () => {
 
   initializers.initClusterMetadataDetectors();
 
-  try {
-    logger.info("🔌 Starting LensProxy");
-    await lensProxy.listen();
-  } catch (error) {
-    dialog.showErrorBox("Lens Error", `Could not start proxy: ${error?.message || "unknown error"}`);
+  logger.info("🔌 Starting LensProxy");
+
+  for (let attempt = 0;; attempt += 1) {
```

Comment:
I think retries should be hidden within `lensProxy.listen()` .. if it cannot find proper port then throw.

Reply:
Sounds good. 

I had initially tried catching the error from the load itself but felt that that was too complicated.